app/controllers/webhook_controller.rb

```diff
@@ -43,6 +45,9 @@ def callback
 
   private
   TEXT_NOT_FOUND = '文字が検出されませんでした。'
+  NOT_AVAILABLE_FORMAT = 'そのファイル形式は文字起こしできません。'
```

Comment:
[IMO]ユーザーフレンドリーにするなら、
`そのファイル形式は文字起こしできません。次の形式で送ってください「 wav」、「.m4a」、「.mp3」`
としてあげると、ユーザーが何をしたらいいかわかるので良いです。

ベタがきでなくても、ダブルクオーテーションと`#{}` を使うと文字列展開ができるので、以下のようにかけるかもしれません。(配列でどうなるかはやってみないとあれですが)
```suggestion
  AVAILABLE_AUDIO_FILE_FORMATS = ['wav', '.m4a', '.mp3']
  NOT_AVAILABLE_FORMAT = "そのファイル形式は文字起こしできません。次の形式で送ってください #{AVAILABLE_AUDIO_FILE_FORMATS}"
```

Reply:
その方がユーザーにとってわかりやすいですね！
変更します！